scania-digital-design-system/tegel
# docs(text-field): type prop to lower case

**Describe pull-request**  
Fix for story file of text field component. Under `type` prop, we had options starting with a capital case which causes an issue when copy-pasting code.

**How to test**  
1. Open the preview link
2. Check Text-field component
3. Toggle type prop from to text to password and back
4. Check Docs tab
5. Make sure that type value is written in small caps

**Suggested test steps**
